config/locales/en/course/assessment/question/text_response.yml

```diff
@@ -7,9 +7,6 @@ en:
             text_response: 'New Text Response Question'
             file_upload: 'New File Upload Question'
             comprehension: 'New Comprehension Question'
-          solution_fields:
-            explanation_hint: 'The explanation to show after the student submits his answer.'
```

Comment:
Port this as well? Please be careful when doing the porting... you should check that we have ported everything before deleting something.

Reply:
done.. alright, will be more careful next time :)